docs/r2/tensorboard_quickstart.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<p>Maybe shift to tfds?</p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/tensorboard/pull/1870/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
dataset, info = tfds.load('mnist', with_info=True, as_supervised=True)
mnist_train, mnist_test = dataset['train'], dataset['test']

def convert_types(image, label):
  image = tf.cast(image, tf.float32)
  image /= 255
  return image, label

mnist_train = mnist_train.map(convert_types).shuffle(10000).batch(32)
mnist_test = mnist_test.map(convert_types).batch(32)

Reply:
Do you know if the [other tutorials](https://github.com/tensorflow/docs/tree/master/site/en/r2) will also shift to tfds? I wanted to keep things as simple as possible and not introduce anything new to people who have only read the basic classification / gradienttape tutorials. Perhaps we can make this change later?

Reply:
Gradient Tape tutorials use tfds if the dataset is available.

https://github.com/tensorflow/docs/tree/master/site/en/r2/tutorials/quickstart

It just makes it easier to do stuff, since you are creating a tf.data.Dataset for training loops it makes sense to shift to that. If it was just using model.fit() then it wouldn't matter.

Reply:
But in the future we will shift all the tutorials to tfds.

Reply:
Oh cool! Thanks for the pointer.

The reason I am hesitant to switch right now is that I think it is more important for this quickstart to align with the beginner quickstart ([here](https://github.com/tensorflow/docs/blob/master/site/en/r2/tutorials/quickstart/quickstart_beginner.ipynb)), which doesn't use tfds right now. What do you think about keeping this change for a little later?

Reply:
SGTM